README.md

```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/drum-IT/multiCLI/badge.svg?branch=master)](https://coveralls.io/github/drum-IT/multiCLI?branch=master)
```

Comment:
do we want to move our default branch to develop to add an extra layer of security?

Reply:
Great idea. I'm going to merge this to wrap up initial integration, and then we can get started on that.